examples/metadata/metadata-volume.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Pod
+apiVersion: v1beta3
+id: kurbernetes-metadata-volume-plugin
+metadata:
+  labels:
+    zone: us-est-coast
+    cluster: test-cluster1
+    rack: rack-22
+  name: kubernetes-metadata-volume-example
+  annotations:
+    build: two
+    builder: john-doe
+spec:
+  containers:
+    - name: client-container
+      image: gcr.io/google_containers/busybox
+      command: ["sh", "-c", "while true; do if [[ -e /etc/labels ]]; then cat /etc/labels; fi; if [[ -e /etc/annotations ]]; then cat /etc/annotations; fi; sleep 5; done"]
+      volumeMounts:
+        - name: metadataplugin
+          mountPath: /etc
+          readOnly: true
+  volumes:
+    - name: metadataplugin
```

Comment:
I don't think "plugin" should be in the volume name, though I realize that just "metadata" could be confusing. How about "podinfo"?